src/tcfv2/sourcepoint.ts

```diff
@@ -28,6 +29,7 @@ export const init = (pubData = {}): void => {
 
 	// invoke callbacks before we receive Sourcepoint events
 	invokeCallbacks();
+	tcfApiEventListener();
 
 	/* istanbul ignore next */
 	window._sp_ = {
```

Comment:
Hmm isn't that doc is for the latest version of sp though? We're on the [legacy one](https://documentation.sourcepoint.com/web-implementation/web-implementation/web-implementation-legacy/sourcepoint-gdpr-and-tcf-v2-support/gdpr-and-tcf-v2-setup-and-configuration_v1.1.3), right?

Reply:
I think by using the new stub, the TCFv2 implementation is effectively no longer legacy.

What’s the [version number] now? It seems you can run `window._sp_.version`.

[version number]: https://documentation.sourcepoint.com/web-implementation/general/version-code

Reply:
It looks like main and this branch are both version 1.2.20

Reply:
It seems to me that we'd need to run
https://cdn.privacy-mgmt.com/unified/wrapperMessagingWithoutDetection.js
rather than 
https://cdn.privacy-mgmt.com/wrapperMessagingWithoutDetection.js
in order to upgrade.
 I can't seem to find an explanation of what has changed between the two versions of the stub but [this](https://github.com/InteractiveAdvertisingBureau/GDPR-Transparency-and-Consent-Framework/blob/master/TCFv2/IAB%20Tech%20Lab%20-%20CMP%20API%20v2.md#requirements-for-the-cmp-stub-api-script) explains the stub script's requirements.

Reply:
Fair enough. Merge away!